lib/model/dob.dart

```diff
@@ -0,0 +1,16 @@
+class Dob {
```

Comment:
Do you think it could be better to give this model the full name of the camp, to make it more easy to understand in the future? in this case it should be DateOfBirth

Reply:
done